api/scpca_portal/test/management/commands/test_create_portal_metadata.py

```diff
@@ -28,7 +30,7 @@ def tearDownClass(cls):
         shutil.rmtree(common.OUTPUT_DATA_PATH, ignore_errors=True)
 
     def assertProjectReadmeContains(self, text, zip_file):
-        self.assertIn(text, zip_file.read("README.md").decode("utf-8"))
+        self.assertIn(text, zip_file.read(README_FILE).decode(ENCODING))
```

Comment:
We will want to match the existing behavior around testing readmes once we merge in #839

Reply:
See at 0c7b5ae416c940b666543f3e32728b3eae7a13f8.

Reply:
Removed the constant `ENCODING` at 0c7b5ae416c940b666543f3e32728b3eae7a13f8  and added the `TODO` comment indicating that after PR #839 is merged into `dev`, readme file format testing will be added (this `assertProjectReadmeContains` method will be removed).